logging/doc.go

```diff
@@ -146,5 +146,33 @@ parent.Timestamp marks the end of the request.)
 You should observe the child log entries grouped under the parent on the console. The
 parent entry will not inherit the severity of its children; you must update the
 parent severity yourself.
+
+# Automatic Trace/Span ID Extraction
+
+You can automatically populate the Trace, SpanID, and TraceSampled fields of an Entry object by providing an [http.Request] object
+within the Entry's HTTPRequest field:
+
+	logging.Entry{
+		HTTPRequest: &logging.HTTPRequest{
+			Request: // Reference to your http.Request here
+		}
+	}
+
+When Entry with an [http.Request] is logged, its Trace, SpanID, and TraceSampled fields may be automatically populated as follows:
+
+ 1. If you are instrumenting your application with [OpenTelemetry],
```

Comment:
It might be helpful to be more specific, and reference go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp, since it would be complicated for users to properly to use this without the instrumentation library itself.